obs-packaging/runtime/debian.rules-template

```diff
@@ -12,7 +12,7 @@ export GOPATH=/usr/src/packages/BUILD/go
 export GOROOT=/usr/src/packages/BUILD/local/go
 export DH_OPTIONS
 
-export DEFAULT_QEMU=qemu-lite-system-x86_64
+export DEFAULT_QEMU=qemu-system-x86_64
```

Comment:
This PR refers to `qemu-vanilla` so I was expecting to see this be `qemu-vanilla-x86_64`? Or by "vanilla", do you mean "the distro-packaged version of QEMU" (I suspect not)?

Also, please can you add a reference to the appropriate issue used for this change (#579?!?)

Reply:
@egernst  James is correct, this is qemu-vanilla the testing job failed because did not found the qemu binary.
Could you please update with qemu-vanillla instead of just qemu.

Reply:
Stepping back -- I hate that name.  If we are seeking to drop qemu-lite, I'd prefer just to say QEMU.  WDYT @jcvenegas @jodh-intel ?

Reply:
Can you clarify? Which name don't you like `qemu-vanilla-system-x86_64`? But if you set `export DEFAULT_QEMU=qemu-system-x86_64` we'll be using the distro-packaged version of qemu which may be different per distro and per architecture.

This PR needs to be clear on whether:

- We're switching to an unmodified version of qemu which we are packaging.
- We're switching the default from `qemu-lite` (which we package) to the distro-packaged version of qemu (which we don't package and may vary as explained above).

Reply:
it can clash with the distro package, what about `kata-qemu` ?

Reply:
Sounds good to me but we probably want some input from folk like @marcov  and @dagrh.

Reply:
IMO it doesn't makes sense to change the binary name, since it's just a build of the official qemu if I see it right.

Given that currently qemu-vanilla package provides a single file `/usr/bin/qemu-vanilla-system-x86_64`, you could:
- change it to be `/usr/lib/kata-containers/qemu-system-x86_64`,
-  use here `DEFAULT_QEMU=/usr/lib/kata-containers/qemu-system-x86_64`


I'd also name the package something like `qemu-4.x.x+kata.[kata-version]`, but that's more a personal preference :)

Reply:
> change it to be /usr/lib/kata-containers/qemu-system-x86_64,

but you have to move everything into `/usr/lib/kata-containers` to dont overwrite the qemu's firmware already installed, isn't it?

Reply:
Where does all the other kata stuff live?  Would just using --prefix=/opt/kata  work?
QEMUs binary names and paths tend to vary a bit between distros as well.

Reply:
Opened issue to track this: https://github.com/kata-containers/packaging/issues/602